### PR TITLE
Followup to SmarSim-Scaling#35

### PR DESCRIPTION
**Pull request by ashao**
For expediency, we had left some comments unresolved in https://github.com/CrayLabs/SmartSim-Scaling/pull/35. This addresses a number of style and requests for comments that were pointed out in that review

### CODE REVIEW

File: imagenet/model_saver.py [hunk 1]
```diff
@@ -10,9 +10,12 @@ def save_model(device: str = "CPU"):
             raise Exception("Requested to create GPU model, but CUDA was not found.")
     # Device's torch name
     device_name = "cuda" if device.startswith("GPU") else "cpu"
-    # model = models.resnet50(pretrained=True)
+
+    # Many of the Imagenet models could be used. shufflenet has about the the same number of
+    # parameters as the ones used in our use cases
     model = models.shufflenet_v2_x0_5(pretrained=True)
-    #model = models.resnet152(pretrained=True)
+    # model = models.resnet50(pretrained=True)  # ~250k parameters
```
Comment (amandarichardsonn) [inline imagenet/model_saver.py:17]: hmmm I think the commented out models.resnet50 code is for users to uncomment -> possibly add this somewhere in the documentation that you can navigate to model_saver.py and uncomment to change

Possibly here https://github.com/CrayLabs/SmartSim-Scaling/tree/master/driverinference under supported inference tests? 

or we can create a ticket bc is it bad that we make the user uncomment code?

File: fortran-inference/inference_scaling_imagenet.F90 [hunk 2]
```diff
@@ -10,6 +10,7 @@ program main
 
 ! Configuration parameters
```
Comment (amandarichardsonn) [inline fortran-inference/inference_scaling_imagenet.F90:11]: Awesome comments! I now understand what this foreign language is doing

File: fortran-inference/inference_scaling_imagenet.F90 [hunk 3]
```diff
@@ -43,36 +44,43 @@ program main
 use_cluster = get_env_var("SS_CLUSTER", .false.)
```
Comment (amandarichardsonn) [inline fortran-inference/inference_scaling_imagenet.F90:44]: I had added a comment in the past that said "Is "SR_LOG_FILE": "srlog.out" and "SR_LOG_LEVEL": "info" needed here?" under "!Retrieve all environment variables" - add if you think this is still applicable!

**Comment by amandarichardsonn** [comment]
Thanks for going back and addressing the previous comments I left! This LGTM just some suggestion comments!

File: imagenet/model_saver.py (hunk 1 above)
Comment (ashao) [reply imagenet/model_saver.py]: Modified this to allow the user to choose which of the torchvision models they wanted

File: fortran-inference/inference_scaling_imagenet.F90 (hunk 3 above)
Comment (ashao) [reply fortran-inference/inference_scaling_imagenet.F90:44]: These are not currently used/set via the Fortran client constructor. In the future, if we decide to use ConfigOptions object, then we could certainly add these then.

**Review by amandarichardsonn — APPROVED** [review]
LGTM! Thanks sm for revisiting the comments and making the changes!

**Pull request merged**